src/components/Tooltip2/Tooltip.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {Children, cloneElement, useCallback, useEffect, useState} from 'react';
+import isFunction from 'lodash/isFunction';
+import isObject from 'lodash/isObject';
+
+import {Popup, PopupPlacement} from '../Popup';
+import {block} from '../utils/cn';
+
+import './Tooltip.scss';
+
+export interface TooltipProps {
+    text: React.ReactNode;
```

Comment:
Let's name it `content`. We usually use this name.

Reply:
Update the name in the latest commit